content/includes/veteran-navigation.html

```diff
@@ -8,13 +8,13 @@
             aria-controls="vetnav-explore"
             role="button"
             aria-haspopup="true"
-            class="vetnav-level1 vetnav-trigger"
```

Comment:
Didn't know what this `.vetnav-trigger` did, so I did a little search. Didn't come up in `src/` anywhere (javascript or sass), and this was the only file in `content/` that it was found in, so I think it's safe to delete.